lib/linalg/Matrix.h

```diff
@@ -946,6 +946,19 @@ class Matrix
     void
     orthogonalize();
 
+    /**
+     * @brief Orthonormalizes the matrix's last column, assuming the previous
+     * columns are already orthonormal.
+     *
+     * By default, the function considers the whole matrix.
```

Comment:
To be a bit more precise, you could say `By default (ncols == -1)`.

Reply:
Done.